### PR TITLE
PAXJDBC-146 introduce the initialSize option in pax-jdbc-pool-narayana

### PR DESCRIPTION
**Pull request by zhfeng**
https://ops4j1.jira.com/browse/PAXJDBC-146

### CODE REVIEW

**Comment by zhfeng** [comment]
@grgrzybek it looks like we have no integration tests for the pax-jdbc-pool-narayana ?

**Comment by grgrzybek** [comment]
that's true... This isn't high-budget / big-team project ;)

**Comment by zhfeng** [comment]
I know and is there any way I can verify this option ?

**Comment by grgrzybek** [comment]
I'm not sure... Adding more tests would be great ;)

**Comment by zhfeng** [comment]
sure, I will take a look.

File: pax-jdbc-itest/src/test/java/org/ops4j/pax/jdbc/test/pool/PoolNarayanaTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    @Inject @org.ops4j.pax.exam.util.Filter("(pool=narayana)(xa=false)")
+    PooledDataSourceFactory pool;
+    
+    @Configuration
+    public Option[] config() {
+        return new Option[] {
+            regressionDefaults(), //
+            poolDefaults(), //
+            mvnBundle("com.h2database", "h2"), //
+            mvnBundle("org.jboss", "jboss-transaction-spi"), //
```
Comment (zhfeng) [inline pax-jdbc-itest/src/test/java/org/ops4j/pax/jdbc/test/pool/PoolNarayanaTest.java]: @grgrzybek I add this itest for the pax-jdbc-pool-narayana but it fails with
```
00:16:15.406 ERROR {Framework Event Dispatcher: org.eclipse.osgi.internal.framework.EquinoxEventPublisher@20140db9} [org.ops4j.pax.exam.nat.internal.NativeTestContainer] : Framework ERROR event org.osgi.framework.FrameworkEvent[source=org.ops4j.pax.jdbc.pool.narayana_1.4.4.SNAPSHOT [27]]
org.osgi.framework.BundleException: Could not resolve module: org.ops4j.pax.jdbc.pool.narayana [27]
  Unresolved requirement: Import-Package: org.jboss.tm

        at org.eclipse.osgi.container.Module.start(Module.java:444) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1620) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1600) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1571) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1514) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230) [org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340) [org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
```
I added
```
mvnBundle("org.jboss", "jboss-transaction-spi")
```
here and it looks like the bundle has not been scanning.
Can you take a look and give some hints please ?

**Comment by zhfeng** [comment]
@grgrzybek can you review this PR ?

File: pax-jdbc-itest/src/test/java/org/ops4j/pax/jdbc/test/pool/PoolNarayanaTest.java (hunk 1 above)
Comment (zhfeng) [reply pax-jdbc-itest/src/test/java/org/ops4j/pax/jdbc/test/pool/PoolNarayanaTest.java]: OK, it looks like we need to use the org.jboss.narayana:narayana-osgi-jta here.

**Comment by zhfeng** [comment]
The itest of PoolNarayanaTest should work now. @grgrzybek do you happy to merge this PR ?

**Pull request merged**